integration/kubernetes/overhead/test_fio/workloads.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fio
+  labels:
+    app: overhead
+spec:
+  runtimeClassName: kata-qemu
```

Comment:
why not making it more generic and have it only as `kata`

Reply:
hum I was testing with `kata-deploy` that installs kata-qemu, it does not have any default like this \cc @egernst